[RFR] Logfile fetch and decode (#106)
pymedphys/pymedphys: changelog.md

    `DicomCollection.from_directory(directory_path)` which would pull all DICOM
    files nested within a directory and sort them into `DicomStudy` objects
    based on their header UIDs.

> - Added CLI commands for a WIP docker server, logfile orchetration, and DICOM

Reviewer: ```suggestion
- Added CLI commands for a WIP docker server, logfile orchestration, and DICOM
```

  editor tools.
- Added a range of xlwings tools that allow the use of PyMedPhys functions
  within Excel

> - Added the rudementry code enabling the pulling of profiles from Mephysto

Reviewer: ```suggestion
- Added rudimentary code to pull profiles from Mephysto
```

Author: Thanks Matt, I think I should find a VSCode extension for spell check...
